Added screenshot details in example section (#150)
yvesgurcan/web-midi-player: .github/ISSUE_TEMPLATE/bug_report.md

*What did you expect to happen?*

> ## Example/ Screenshots

Reviewer: I would like to suggest a simple fix related to typography:
```suggestion
## Example/Screenshots
```

> *Please provide relevent screenshots  as it not only allows you to help explain the problem more clearly but also allows the reviewer to visually see the bug or its effect*

Reviewer: Hi Hemant, thank you so much for your contribution! How do you feel about changing the text you wrote a little bit?

```suggestion
*Please provide screenshots to help reviewers  to visualize the bug and how it manifests.*
```
